### ISSUE TITLE
Importing Anthropic SDK crashes Vercel Edge Runtime on non-NextJS projects 

### ISSUE THREAD

**Issue by dqbd** [anthropics/anthropic-sdk-typescript#65]
When importing `@anthropic-ai/sdk@0.5.4` in Vercel Edge Runtime, the following exception occurs:

```
TypeError: Cannot read properties of undefined (reading 'custom')
    at (node_modules/object-inspect/index.js:69:0)
    at ([native code])
    at (node_modules/side-channel/index.js:5:0)
    at ([native code])
    at (node_modules/qs/lib/stringify.js:3:0)
    at ([native code])
    at (node_modules/qs/lib/index.js:3:0)
    at ([native code])
    at (node_modules/@anthropic-ai/sdk/core.js:91:0)
    at ([native code])
```

It does seem like the issue stems from the usage of `qs`, which is unsupported by Vercel Edge Runtime. 

Reproducible repository: https://github.com/dqbd/anthropic-vercel-edge

cc: @jacoblee93

**Comment by RobertCraigie** [comment]
@dqbd how are you building/deploying to Vercel Edge? It doesn't seem like there's a build command in your reproduction repo.

I can successfully deploy to Vercel Edge with an API route that looks like this:
```ts
import { NextRequest, NextResponse } from 'next/server';
import { Anthropic } from '@anthropic-ai/sdk';

const anthropic = new Anthropic();

export const config = {
  runtime: 'edge',
  unstable_allowDynamic: [
    // Note: this is due to a redundant error in Next.js that has
    // been fixed in the latest version
    '/node_modules/function-bind/**',
  ],
};

export default async (request: NextRequest) => {
  console.log(anthropic);
  return NextResponse.json({ foo: true });
};
```

<details>
<summary>Extra Details</summary>

`package.json`:
```json
{
  "name": "vercel-edge-test",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "vercel": "vercel"
  },
  "dependencies": {
    "@anthropic-ai/sdk": "^0.5.4",
    "ai": "2.1.9",
    "next": "13.4.6",
    "react": "18.2.0",
    "react-dom": "18.2.0"
  },
  "devDependencies": {
    "@types/node": "20.3.3",
    "@types/react": "18.2.13",
    "@types/react-dom": "18.2.6",
    "edge-runtime": "^2.4.3",
    "typescript": "5.1.3",
    "vercel": "^30.2.3"
  }
}
```

`tsconfig.json`
```json
{
  "compilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true,
    "paths": {
      "~/*": ["./src/*"]
    }
  },
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
  "exclude": ["node_modules"]
}
```
</details>

**Comment by dqbd** [comment]
@RobertCraigie We're sending the source code of the Vercel Edge Function directly, deferring the bundling on Vercel. We're also not using Next.JS or Webpack for building functions at the moment. 

EDIT: It does seem like NextJS does bundle the edge function before sending it to Vercel and the package thus works on latest Next.JS. Updated the title.

**Comment by rattrayalex** [comment]
Thanks @dqbd – we're talking to Vercel about ways to resolve this.

**Comment by rattrayalex** [comment]
This should be resolved as of 0.3.7, which is published on GitHub and should be live on npm shortly!
